.github/workflows/cypress.yml

```diff
@@ -7,6 +7,10 @@ on:
 jobs:
```

Comment:
Can we also move the Lint step here? Shouldn't we give up the `.travis.yml` completely then? With this PR there is no reason to keep it. 

Furthermore, you have at least remove the deploy step: https://github.com/Fewwy/insights-advisor-frontend/blob/ghactions/.travis.yml#L15. Otherwise, you have a double call of the deploy script.

Reply:
Removed the travis.yml and renamed the cypress.yml to deploy.yml + updated it